libcudacxx/docs/extended_api/memory_resource/resource_ref.md

```diff
@@ -0,0 +1,49 @@
+---
+grand_parent: Extended API
+parent: memory_resource
+nav_order: 3
+---
+
+### `resource_ref`: a type-constrained resource wrapper
+
+With the property design shown above a library has flexibility in checking constraints and querying custom properties. However, there is also a cost in providing function templates for a potentially wide range of inputs. Depending on the number of different memory resources, both compile time and binary size might increase considerably.
```

Comment:
problem: now that properties are in a separate markdown file, "shown above" doesn't make sense. In general directional references like "above" and "below" should be avoided. 

suggestion: Use direct section references.

Reply:
I would use "described", not "depicted", since there are no pictures. But otherwise this approach looks good.